src/assets/linkPreview.js

```diff
@@ -0,0 +1,250 @@
+/* eslint-disable no-console */
+/* eslint-disable no-undef */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable no-unused-vars */
```

Comment:
I would prefer to not disable these rules for the whole file and instead manually disable them for specific lines:
`// eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars`
and
`// eslint-disable-next-line no-undef`